libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -1440,6 +1429,24 @@ void AP_BLHeli::read_telemetry_packet(void)
     }
 }
 
+// Log all of the ESC telemetry at
+void AP_BLHeli::log_telemetry(void)
+{
+    AP_Logger *logger = AP_Logger::get_singleton();
+    if (logger && logger->logging_enabled()) {
```

Comment:
Do early returns, please.

Reply:
😄 this is literally copy and paste of @tridge 's code